cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Does this file need to exist?

Reply:
It'll be used in the next PR to make cypress configurable for multiple envs

Reply:
It maybe should be deleted, and just added in iteratively when it is needed.

Reply:
This is the same as the directories `fixtures`,`support` and `plugins` in that if it doesn't exist when you run cypress it creates it - therefore can't be deleted